core/domain/classifier_domain.py

```diff
@@ -153,10 +153,11 @@ def validate(self):
                 self.algorithm_id)
         utils.require_valid_name(
             self.algorithm_id, 'the algorithm id')
-        list_algorithms = []
-        for interaction_type in feconf.INTERACTION_CLASSIFIER_MAPPING.values():
-            list_algorithms.append(interaction_type['algorithm_id'])
-        if self.algorithm_id not in list_algorithms:
+        algorithms = []
```

Comment:
@pranavsid98 oh I'm really sorry but I should have mentioned algorithm_ids. That sounds more appropriate, doesn't it?